webapp/components/interop.js

```diff
@@ -11,17 +11,29 @@ import '../node_modules/@polymer/paper-input/paper-input.js';
 import '../node_modules/@polymer/polymer/lib/elements/dom-if.js';
 import { html, PolymerElement } from '../node_modules/@polymer/polymer/polymer-element.js';
 import {CountUp} from 'https://unpkg.com/countup.js@2.0.8/dist/countUp.js';
-import PARAMS_BY_YEAR from '../static/interop-data.json' assert {type: 'json'};
 
 // InteropDataManager encapsulates the loading of the CSV data that backs
 // both the summary scores and graphs shown on the Interop dashboard. It
 // fetches the CSV data, processes it into sets of datatables, and then caches
 // those tables for later use by the dashboard.
 class InteropDataManager {
   constructor(year) {
-    // prepare all year-specific info for reference.
     this.year = year;
-    const yearInfo = PARAMS_BY_YEAR[year];
+    // The data is loaded when the year data is obtained and the csv is loaded and parsed.
+    this._dataLoaded = this.fetchYearData()
+    // The year data is needed for parsing the csv.
+      .then(async() => {
+        await load();
+        return await Promise.all([this._loadCsv('stable'), this._loadCsv('experimental')]);
```

Comment:
```suggestion
        return Promise.all([this._loadCsv('stable'), this._loadCsv('experimental')]);
```
Is `await` here needed?

Reply:
IIUC, `this.fetchYearData()` is called synchronously at the time of `this._dataLoaded` creation, so we can guarantee that fetchYearData is fetched before use.

Reply:
I think you're right, I've made the change 👍